pims/image_reader.py

```diff
@@ -6,25 +6,30 @@
 from pims.base_frames import FramesSequence, FramesSequenceND
 from pims.frame import Frame
 
-# skimage.io.plugin_order() gives a nice hierarchy of implementations of imread.
-# If skimage is not available, go down our own hard-coded hierarchy.
+# If scikit-image is not available, use matplotlib (with a warning) instead.
+import warnings
 try:
     from skimage.io import imread
 except ImportError:
     try:
         from matplotlib.pyplot import imread
+        # imread() works differently between scikit-image and matplotlib.
+        # We don't require users to have scikit-image, 
+        # but if we fall back to matplotlib, make sure the user 
+        # is aware of the consequences.
+        skimage_preferred = "pim's image_reader.py could not find scikit-image. Falling back to matplotlib's imread(), which uses floats instead of integers. This may break your scripts." \
```

Comment:
Could you wrap the lines to 79 characters? And a typo in the fist word: `pim` should be `PIMS`.